tools/checkpatch.sh

```diff
@@ -84,14 +84,14 @@ check_patch() {
     fail=1
   else
     git apply $1
-    diffs=`cat $1`
+    diffs=``
     check_ranges <<< "$diffs"
     git apply -R $1
   fi
 }
 
 check_commit() {
-  diffs=`git show $1`
+  diffs=`git show --no-renames $1`
```

Comment:
This can just be `git diff $1` which will resolve the issue with renames.  `show` was the wrong command to use here.

Reply:
Ok, I confirm it works. Although I have to be standing on nuttx/. From nuttx/tools it does not work.

Reply:
The check failed, for some reason it seems to be getting a strange set of commits.